src/features/embalm/stepContent/hooks/useArchaeologistList.ts

```diff
@@ -0,0 +1,89 @@
+import { useCallback } from 'react';
+import {
+  deselectArchaeologist,
+  selectArchaeologist,
+  SortDirection,
+  setDiggingFeesSortDirection,
+  setArchAddressSearch,
+} from 'store/embalm/actions';
+import { useDispatch, useSelector } from 'store/index';
+import { Archaeologist } from 'types/index';
+import { useLoadArchaeologists } from '../hooks/useLoadArchaeologists';
+import { orderBy } from 'lodash';
+
+export function useArchaeologistList() {
+  useLoadArchaeologists();
+  const dispatch = useDispatch();
+  const {
+    archaeologists,
+    selectedArchaeologists,
+    diggingFeesSortDirection,
+    diggingFeesFilter,
+    archAddressSearch,
+  } = useSelector(s => s.embalmState);
+
+  // TODO: It doesn't make sense to implement pagination any further until we are loading real archaeologists
+  const onClickNextPage = useCallback(() => {
+    // Temporary console log
+    console.log('Will implement pagination when we are loading real archaeologists');
+  }, []);
+
+  const onClickPrevPage = useCallback(() => {
+    // Temporary console log
+    console.log('Will implement pagination when we are loading real archaeologists');
+  }, []);
+
+  const handleCheckArchaeologist = useCallback(
+    (archaeologist: Archaeologist) => {
+      if (selectedArchaeologists.includes(archaeologist)) {
+        dispatch(deselectArchaeologist(archaeologist.profile.archAddress));
+      } else {
+        dispatch(selectArchaeologist(archaeologist));
+      }
+    },
+    [dispatch, selectedArchaeologists]
+  );
+
+  function onClickSortDiggingFees() {
```

Comment:
I entirely dislike my implementation of this logic. Will entertain any suggestions.

Reply:
The chained ternary operator used here is pretty confusing. This might be a good place for a `switch case`. A mapping of `SortDirection` to `SortDirection` would also work, but `switch case` is probably the most readable.